### PR TITLE
Deploy Crosschain token scripts

### PR DESCRIPTION
**Pull request by ququzone**

### CODE REVIEW

File: scripts/addresses.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "proxy_token_list": "0x6ccf305a21defff295e616ba5aa423eb563fc8db",
+        "witness_list": "0xbe441d37c48aabac5116fce76bbc45a013f6d44d",
+        "token_cashier": "0x44074576e015bfd4f93f074671bd5a2a55c5d9c5",
+        "transfer_validator": "0x7987aaf615b0f749f12da7d4d6e2b1eff4bfbe1b"
+    },
+    "bsc": {
+        "token_safe": "0xFBe9A4138AFDF1fA639a8c2818a0C4513fc4CE4B",
+        "minter_pool": "0xf72CFb704d49aC7BB7FFa420AE5f084C671A29be",
+        "standard_token_list": "0x0d793F4D4287265B9bdA86b7a4083193E8743b34",
+        "proxy_token_list": "", // mintable token list?: 0xa6ae9312D0AA3CC74d969Fcd4806d7729A321EE3
```
Comment (CoderZhi) [inline scripts/addresses.js:36]: 0xa6ae9312D0AA3CC74d969Fcd4806d7729A321EE3

File: scripts/addresses.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        "witness_list": "0x8119411F5A78F73784A1B87dE43d452DA4A1EE3F",
+        "token_cashier": "0x797f1465796fd89ea7135e76dbc7cdb136bba1ca",
+        "transfer_validator": "0x116404F86e97846110EA08cd52fC2882d4AD3123",
+        "router": "0x2a26feDe01BfA03E65cD3C68f421cAB83648391a"
+    },
+    "polygon": {
+        "token_safe": "0xA239F03Cda98A7d2AaAA51e7bF408e5d73399e45",
+        "minter_pool": "0x12af43ef94B05A0a3447A05eEE629C7D88A30a5f",
+        "standard_token_list": "0xDe9395d2f4940aA501f9a27B98592589D14Bb0f7",
+        "proxy_token_list": "", // mintable token list?: 0xC8DC8dCDFd94f9Cb953f379a7aD8Da5fdC303F3E
```
Comment (CoderZhi) [inline scripts/addresses.js:46]: 0xC8DC8dCDFd94f9Cb953f379a7aD8Da5fdC303F3E

**Review by CoderZhi — APPROVED** [review]

**Comment by guo** [comment]
please also help fix the unit tests.

**Comment by CoderZhi** [comment]
it is the version problem, not sure how to fix

**Pull request merged**